src/schema/sources.ts

```diff
@@ -312,6 +318,26 @@ export const typeDefs = /* GraphQL */ `
       image: Upload
     ): Source! @auth
 
+    """
+    Set the source member's current role
+    """
+    updateMemberRole(
```

Comment:
Maybe we should name it `changeMemberRole` as it does both update and demote?

Reply:
The update is applicable to demote and promote I believe. I mean, it just simply means change.